client/components/navigation/Navbar.tsx

```diff
@@ -1,39 +1,46 @@
+import React, { useState } from 'react';
+import {
+  chakra,
+  Link,
+  LinkOverlay,
+  Flex,
+  HStack,
+  Image,
+  Button,
+  VStack,
+} from '@chakra-ui/react';
 
-import React from 'react';
-import { chakra, Link, LinkOverlay, Flex, HStack, Image } from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
 
-import {ButtonBlack} from "../../styles/ui-components/Chakra-Button";
+import { ButtonBlack } from '../../styles/ui-components/Chakra-Button';
 
 function Navbar() {
+  const [isHamburgerOpen, setIsHamburgerOpen] = useState(false);
```

Comment:
Chakra comes with the [`useDisclosure` hook](https://chakra-ui.com/docs/hooks/use-disclosure) which makes it easy to manage the open/closed state of a thing!

```tsx
const {isOpen, onOpen, onClose, onToggle} = useDisclosure();
onOpen()
onClose()
onToggle()

// or (for more clear name/purpose)

const hamburger = useDisclosure();
hamburger.onOpen()
hamburger.onClose()
hamburger.onToggle()
```